hasura/metadata/databases/default/tables/public_farcaster_accounts.yaml

```diff
@@ -4,13 +4,7 @@ table:
 object_relationships:
   - name: profile_public
     using:
-      manual_configuration:
-        column_mapping:
-          profile_id: id
-        insertion_order: null
-        remote_table:
-          name: profiles_public
-          schema: public
+      foreign_key_constraint_on: profile_id
```

Comment:
I feel like we just flipped this in the other dir - whats going on here? I might have messed this up

Reply:
keeping what we have on main.